bld.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # bld.sh - build the programs on Linux.
 
-source ~/lbm.sh
+source ../../lbm.sh
```

Comment:
Ah, I checked in that change and didn't tell anybody. No problem with changing it back. I'll re-visit in the future.

Reply:
Oh, and it should probably be "../lbm.sh". Omit the extra "../".